virttest/utils_test/libvirt.py

```diff
@@ -1507,38 +1513,50 @@ def cleanup_dest_vm(self, vm, srcuri, desturi):
         # Set connect uri back to local uri
         vm.connect_uri = srcuri
 
-    def check_vm_state(self, vm, state='paused', uri=None):
+    def check_vm_state(self, vm_name, state='paused', uri=None):
         """
         checks whether state of the vm is as expected
 
-        :param vm: VM Object
+        :param vm_name: VM name
         :param state: expected state of the VM
         :param uri: connect uri
 
         :return: True if state of VM is as expected, False otherwise
         """
-        if not virsh.domain_exists(vm.name, uri=uri):
+        if not virsh.domain_exists(vm_name, uri=uri):
             return False
-        result = virsh.domstate(vm.name, uri=uri)
+        result = virsh.domstate(vm_name, uri=uri)
         vm_state = results_stdout_52lts(result).strip()
         return vm_state.lower() == state.lower()
 
-    def wait_for_migration_start(self, vm, state='paused', uri=None, timeout=60):
+    def wait_for_migration_start(self, vm, state='paused', uri=None,
+                                 migrate_options=None, timeout=60):
```

Comment:
let us default `make migrate_options=""`, otherwise we could hit error in L1554
```python
AttributeError: 'NoneType' object has no attribute 'count'
```

Reply:
Thanks. Updated.